.github/workflows/publish-action.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish Package to NPM
```

Comment:
Would you mind extracting this workflow to its own PR?  That way, tweeting the CD workflow doesn't block merging this PR :)

Reply:
I'm sorry for the trouble, this was MB cuz I created a single issue for both subjects

Reply:
Yeah, no problem. Going to assign to you the new PR! And already remove from here!